extensions/amp-accordion/1.0/accordion.js

```diff
@@ -109,11 +110,12 @@ function AccordionWithRef(
   );
 
   const toggleExpanded = useCallback(
-    (id) => {
+    (id, opt_expand) => {
       setExpandedMap((expandedMap) => {
+        const newExpanded = opt_expand || !expandedMap[id];
```

Comment:
It's best to keep the structure to the classical toggle, which is either the `opt_expand` is `undefined` and we compute it, or it's `true|false` and then we use the value as is. In simple terms this becomes:

```
const newExpanded = opt_expand ?? !expandedMap[id];
```

I.e. `??` instead of `||`.

Reply:
Thanks, yes, changed to `??` to have logic as classic toggle.  Thanks!